plugins/catalog-backend/package.json

```diff
@@ -32,7 +32,7 @@
     "express-promise-router": "^3.0.3",
     "fs-extra": "^9.0.0",
     "git-url-parse": "^11.4.0",
-    "knex": "^0.21.1",
+    "knex": "^0.21.6",
```

Comment:
There was an noisy warning during testing, but its fixed with the update.

There is also a warning from ts-jest that could be resolved with an update, but that update fails somewhere else

Reply:
I could fix the ts-jest problem too, the remaining issues are unrelated to my changes?

Reply:
The ts-jest problem ist fixed to 😉

Reply:
Had to revert both changes, looks like they have issues in tests somewhere else 🤔

Reply:
I will keep the knex update, but the ts-jest one still has problems.